add a faq document for TIC (#1564)
Samsung/tizen-docs: docs/platform/reference/tic-faq.md

<!-- omit in toc -->
# TIC Frequently Asked Questions

- [What is for the easy mode and the advanced mode ?](#what-is-for-the-easy-mode-and-the-advanced-mode-)
- [What is the building block ?](#what-is-the-building-block-)
- [How to get a recipe for TIC ?](#how-to-get-a-recipe-for-tic-)
- [How to add my application to the Tizen binary ?](#how-to-add-my-application-to-the-tizen-binary-)
- [How to add my platform packages to the Tizen binary ?](#how-to-add-my-platform-packages-to-the-tizen-binary-)
- [Can I find artifacts of TIC in my host PC ?](#can-i-find-artifacts-of-tic-in-my-host-pc-)

> ## What is for the easy mode and the advanced mode ?

Reviewer: Same here, please consider revising the sentence.
```suggestion
## What are the "Easy" and "Advanced" modes ?
```

- If you select the easy mode, and click the `apply changes` button, the configurations for building binary are same as the [Tizen public binaries](http://download.tizen.org/releases/milestone/tizen/unified/latest/images/standard/).
  So, you can make same binaries as the Tizen public binaries.

- Regarding to the advanced mode, you can combine packages as you want by selecting building blocks. At first, you can select the architecture in the `Setting` tab,
  And then, you can select building block packages in order by category and step by step in `Packages` tab.

## What is the building block ?

- The building block is the package which defines a set of combined packages for specific propose.
  You can see pre-defined building blocks and the structure of them in [the repository of building blocks](https://git.tizen.org/cgit/tools/building-blocks/tree/?h=tizen).

>   You also can define by yourself your own blocks by creating `.inc` files and add them to [building-block.spec](https://git.tizen.org/cgit/tools/building-blocks/tree/packaging/building-blocks.spec?h=tizen) file

Reviewer: ```suggestion
  You can also define your own blocks by creating `.inc` files and adding them to the [building-block.spec](https://git.tizen.org/cgit/tools/building-blocks/tree/packaging/building-blocks.spec?h=tizen) file.
```

## How to get a recipe for TIC ?

> - You can get the reference recipe by clicking the `Export` button in the advanced mode in the `Setting` tab, or by clicking the `Recipe` button in the `Images` tab if you have binaries already built.

Reviewer: Sentence is a little ambiguous, by "binaries already built" do you mean "pre-built binaries"?

```suggestion
- You can get the reference recipe by clicking the `Export` button in "Advanced" mode in the `Settings` tab, or by clicking the `Recipe` button in the `Images` tab if you have binaries already built.
```

## How to add my applications to the Tizen binary ?

> - You can add your application packages(`tpk` packages) in `setting` tab in both of the easy and advanced mode by clicking the `import` button and selecting the radio button for `tpk` file.

Reviewer: ```suggestion
- You can add your application packages (`tpk` packages) in the `Settings` tab in both "Easy" and "Advanced" mode by clicking the `Import` button and selecting the radio button for `tpk` file.
```

## How to add my platform packages to the Tizen binary ?

> - You can your platform module packages(`rpm` packages) in `setting` tab in the advanced mode, by clicking the `import` button and selecting the radio button for `rpm` file . But, to add your `rpm` packages to the binary, you have to check them in the tree view of the `Packages` tab.

Reviewer: ```suggestion
- You can add your platform module packages (`rpm` packages) in the `Settings` tab in "Advanced" mode, by clicking the `Import` button and selecting the radio button for `rpm` file. However, in order to add your `rpm` packages to the binary, you have to check them in the tree view of the `Packages` tab.
```

## Can I find artifacts of TIC in my host PC ?

> - You can find them in the `tif-artifaces` directory in your home directory, if you use `docker-compose`(https://s3-us-west-1.amazonaws.com/tizenschool/257/docker-compose.yaml) file provided by us.

Reviewer: Should it be "tif-artifaces" or "tif-artifacts"? The heading says "artifacts" so I'm uncertain here. If the correct term is "artifaces", should the heading be changed as well?
```suggestion
- You can find them in the `tif-artifaces` directory in your home directory, if you use the `docker-compose` file  (https://s3-us-west-1.amazonaws.com/tizenschool/257/docker-compose.yaml) provided by us.
```

<!-- omit in toc -->

> # TIC Frequently Asked Questions

Reviewer 2: I see that the FAQ page format is different from [https://docs1.stg.tizen.org/staging/1564/platform/reference/gbs/gbs-faq/](https://docs1.stg.tizen.org/staging/1564/platform/reference/gbs/gbs-faq/) and MIC Faq, however I like this page's format so I guess we can keep it as is.

- [What is the building block?](#what-is-the-building-block)
- [What are the "Easy" and "Advanced" mode?](#what-are-the-easy-and-advanced-mode)
- [How to get a recipe for TIC?](#how-to-get-a-recipe-for-tic)
- [How to add my applications to the Tizen binary?](#how-to-add-my-applications-to-the-tizen-binary)
- [How to add my platform packages to the Tizen binary?](#how-to-add-my-platform-packages-to-the-tizen-binary)
- [Can I find artifacts of TIC in my host PC?](#can-i-find-artifacts-of-tic-in-my-host-pc)

## What is the building block?

- The building block is the package that defines a set of combined packages for a specific purpose.
  You can see pre-defined building blocks and their structure in [the repository of building blocks](https://git.tizen.org/cgit/tools/building-blocks/tree/?h=tizen).

>   You can also define by yourself your own blocks by creating `.inc` files and adding them to the [building-block.spec](https://git.tizen.org/cgit/tools/building-blocks/tree/packaging/building-blocks.spec?h=tizen) file.

Reviewer: Just noticed this, the "by yourself" part is redundant so suggesting you to removing it.
```suggestion
  You can also define your own blocks by creating `.inc` files and adding them to the [building-block.spec](https://git.tizen.org/cgit/tools/building-blocks/tree/packaging/building-blocks.spec?h=tizen) file.
```

## What are the "Easy" and "Advanced" mode ?

- If you select "Easy" mode, you can create exactly the same binaries as the officially released [Tizen binaries](http://download.tizen.org/releases/milestone/tizen/unified/latest/images/standard/). Also, you can create your own binary based on the official one by importing your own packages in this mode.

> - On the other hand, in "Advanced" mode, you can customize your own binary from the scratch. At first, you can select the architecture in the `Settings` tab,

Reviewer: You don't need a defining article here.
```suggestion
- On the other hand, in "Advanced" mode, you can customize your own binary from scratch. At first, you can select the architecture in the `Settings` tab,
```

>   then, you can select the building block packages one by one in the `Packages` tab. 

Reviewer: No comma necessary here.
```suggestion
  then you can select the building block packages one by one in the `Packages` tab. 
```

>   We recommand you select a building block in the `Preset` category first, and the `Domain` category later.

Reviewer: Spelling error needs to be addressed.
```suggestion
  We recommend you select a building block in the `Preset` category first, and the `Domain` category later.
```

  It will help you to select essential packages.

## How to get a recipe for TIC ?

- You can get the reference recipe by clicking the `Export` button in "Advanced" mode in the `Settings` tab, or by clicking the `Recipe` button in the `Images` tab if you have binaries already built.

## How to add my applications to the Tizen binary ?

- You can add your application packages (`tpk` packages) in the `Settings` tab in both "Easy" and "Advanced" mode by clicking the `Import` button and selecting the radio button for `tpk` file.

## How to add my platform packages to the Tizen binary ?

> - You can add your platform module packages (`rpm` packages) in the `Settings` tab in "Advanced" mode, by clicking the `Import` button and selecting the radio button for `rpm` file . However, in order to add your `rpm` packages to the binary, you have to check them in the tree view of the `Packages` tab.

Reviewer: ```suggestion
- You can add your platform module packages (`rpm` packages) in the `Settings` tab in "Advanced" mode. Just click the `Import` button and select the radio button for `rpm` file. However, in order to add your `rpm` packages to the binary, you have to check them in the tree view of the `Packages` tab.
```

## Can I find artifacts of TIC in my host PC?

> - You can find them in the `tic-artifacts` directory in your home directory, if you use the [docker-compose](https://s3-us-west-1.amazonaws.com/tizenschool/257/docker-compose.yaml) file provided by us.

Reviewer: The comma is not necessary here.
```suggestion
- You can find them in the `tic-artifacts` directory in your home directory if you use the [docker-compose](https://s3-us-west-1.amazonaws.com/tizenschool/257/docker-compose.yaml) file provided by us.
```

> - You can find them in the `tic-artifacts` directory in your home directory, if you use the [`docker-compose`](https://s3-us-west-1.amazonaws.com/tizenschool/257/docker-compose.yaml) file provided by us.

Reviewer: The text "docker-compose" should show up as a link only, it's difficult to notice when the text is highlighted as well. 
```suggestion
- You can find them in the `tic-artifacts` directory in your home directory, if you use the [docker-compose](https://s3-us-west-1.amazonaws.com/tizenschool/257/docker-compose.yaml) file provided by us.
```
